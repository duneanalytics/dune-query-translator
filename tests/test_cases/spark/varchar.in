select cast('123456789' as varchar(5))