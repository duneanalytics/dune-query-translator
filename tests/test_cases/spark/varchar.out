SELECT
  TRY_CAST('123456789' AS VARCHAR)